domain/cloud/state/state.go

```diff
@@ -340,9 +340,12 @@ SELECT cloud.uuid, cloud.name, cloud_type_id,
        auth_type.type, cloud_type.type,
        COALESCE((SELECT true
         FROM model_metadata
+		INNER JOIN user
+        ON user.uuid = model_metadata.owner_uuid
         WHERE cloud_uuid = cloud.uuid
-        AND name = ?
-        AND owner_uuid = ?), false) AS controller_cloud
+        AND model_metadata.name = ?
+        AND user.name = ?
+        AND user.removed = false), false) AS controller_cloud
```

Comment:
We don't want to filter on remove = false.
A model is owned by a single user, whether removed or not. But regardless, this is to set the "is controller cloud" value. Whether that is true or not is totally orthogonal to the state of the user who owns the controller model. And given that the owner of the controller model is "admin" and cannot be removed, this will never be false anyway.

Reply:
Also,we can't just make this change without fixing the other bits - the set up of the schema, the setting of the uuid etc. I'd leave the cloud owner changes to a separate PR